code/game/gamemodes/cm_initialize.dm

```diff
@@ -687,9 +687,9 @@ Additional game mode variables.
 	return TRUE
 
 /// Pick and setup a queen spawn from landmarks, then spawns the player there alongside any required setup
-/datum/game_mode/proc/pick_queen_spawn(datum/mind/ghost_mind, hivenumber = XENO_HIVE_NORMAL)
+/datum/game_mode/proc/pick_queen_spawn(mob/new_player/NP, hivenumber = XENO_HIVE_NORMAL)
```

Comment:
What is passed here isn't a `/mob/new_player`. Its a `/mob/living/carbon/human`
![image](https://github.com/cmss13-devs/cmss13/assets/76988376/fbfcd0fb-da9d-4b2a-9f82-691ccdc9f68a)
Also https://github.com/cmss13-devs/cmss13/blob/master/.github/guides/STYLES.md#dont-use-abbreviations

```suggestion
/datum/game_mode/proc/pick_queen_spawn(mob/player, hivenumber = XENO_HIVE_NORMAL)
```